src/index.js

```diff
@@ -0,0 +1,28 @@
+const Airport = require(`./Airport`);
+const Plane = require(`./Plane`);
+
+const Voyager = new Plane('Voyager');
```

Comment:
Instead of defining plane instances in the index, perhaps could have had a class/object with an array of 'active planes' which defines and holds plane objects independently of airport instances. This would simplify the test code as well as the index as no need to define plane instances for each new airport or plane landing.